models/OpenAPI/Parser.cfc

```diff
@@ -202,23 +276,23 @@ component name="OpenAPIParser" accessors="true" {
 			//Files receive a parser reference
 			if( left( FilePath, 4 ) == 'http'  ){
 
-				var ReferenceDocument = Wirebox.getInstance( "OpenAPIParser@SwaggerSDK" ).init(  $ref );
+				ReferenceDocument = Wirebox.getInstance( "OpenAPIParser@SwaggerSDK" ).init(  $ref );
 
 			} else if( len( FilePath ) && fileExists( getDirectoryFromPath( getBaseDocumentPath() ) &  FilePath )){
 
-				var ReferenceDocument = Wirebox.getInstance( "OpenAPIParser@SwaggerSDK" ).init(  getDirectoryFromPath( getBaseDocumentPath() ) & $ref );
+                ReferenceDocument = Wirebox.getInstance( "OpenAPIParser@SwaggerSDK" ).init(  getDirectoryFromPath( getBaseDocumentPath() ) & FilePath );
```

Comment:
This line change actually, broke the `$ref` inheritance functionality.  I've reverted it back to use `$ref`